### ISSUE TITLE
Implement support for constants

### ISSUE THREAD

**Issue by bobbimanners** [bobbimanners/EightBall#23]
Would be nice to be able to define constants and use them in constant expressions.

For example
```
const size=10
word A[size*size] = {}
word i=0
for i=1:size
 A[i]=1
endfor
```
However we are super short of memory so we may need to implement this in a devious way.

**Comment by bobbimanners** [comment]
Maybe this is a 'cheaper' way to implement this feature:
```
word _size=10
```
Currently variable names must begin with alpha character.  We could use a leading underscore to mark a constant.

**Comment by bobbimanners** [comment]
Maybe with trailing _ too?
```
word _size_=10; ' Looks better?
```

**Comment by bobbimanners** [comment]
Implemented in v0.66 (with nicer syntax - see docs!)
